modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -184,11 +183,8 @@ public void apply(Project project) {
 
 		TargetPlatformRootProjectConfigurator.INSTANCE.apply(project);
 
-		CreateTokenTask createTokenTask = _addTaskCreateToken(
```

Comment:
If we stop adding this createToken task, it will break developer's builds that have this in their build.gradle
```
createToken {
//
}
```
So we need to keep adding the task here even though it is deprecated.

Reply:
hey greg, i remove it base on seiphon request. for user current build.gradle he will still use old version workspace?plugin. The new version workspace plug in will not include the createToken task and it will only apply to new version template.

Reply:
hi greg, this change will only apply to new project template. Do we need to make it  backwards compatibility with user's created project or just apply to new project?